packages/permission-controller/src/rpc-methods/revokePermissions.test.ts

```diff
@@ -40,7 +44,7 @@ describe('revokePermissions RPC method', () => {
           snap_dialog: {},
         },
       ],
-    });
+    })) as JsonRpcSuccess<null>;
```

Comment:
GitHub is failing me here, but perhaps similar as above, where we can use `assertIsJsonRpcSuccess(response)` to do this instead of a type assertion?